src/main/java/io/nats/service/ServiceBuilder.java

```diff
@@ -129,16 +128,6 @@ public ServiceBuilder infoDispatcher(Dispatcher infoDispatcher) {
         return this;
     }
 
-    /**
-     * Optional dispatcher for the schema service
-     * @param schemaDispatcher the dispatcher
-     * @return the ServiceBuilder
-     */
-    public ServiceBuilder schemaDispatcher(Dispatcher schemaDispatcher) {
-        this.schemaDispatcher = schemaDispatcher;
-        return this;
-    }
-
```

Comment:
Again, because of semver and not wanting to major version, you can deprecate this and make it a no-op, but unfortunately cannot remove a public api.